cylc/flow/cfgspec/workflow.py

```diff
@@ -657,28 +704,29 @@
             other namespaces inherit from it, or a task if no others inherit
             from it.
 
-            Names may not contain colons (which would preclude use of
-            directory paths involving the registration name in ``$PATH``
-            variables). They may not contain the ``.`` character (it will be
-            interpreted as the namespace hierarchy delimiter, separating
-            groups and names -huh?).
+            Names may not contain ``:`` or ``.``.
+
+            ..note::
+
+               ``:`` would stop cylc adding paths using task name to ``$PATH``.
+               ``.`` would be interpreted as a namespace delimeter
 
             legal values:
 
             - ``[foo]``
             - ``[foo, bar, baz]``
 
             If multiple names are listed the subsequent settings apply to
-            each.
+            all.
 
             All namespaces inherit initially from ``root``, which can be
             explicitly configured to provide or override default settings for
             all tasks in the workflow.
         '''):
             Conf('platform', VDR.V_STRING)
             Conf('inherit', VDR.V_STRING_LIST, desc='''
-                A list of the immediate parent(s) this namespace inherits
-                from. If no parents are listed ``root`` is assumed.
+                A list of the immediate parent(s) of this namespace.
+                If no parents are listed default is ``root``.
```

Comment:
```suggestion
                If no parents are listed the task will inherit from ``[[root]]``.
```

Reply:
I think "If no parents are listed default is ``root``" is simpler

Reply:
Yes, because the task will still "inherit from root" (as well) even if other parents are specified.